### PR TITLE
Remove n^2 rendering problem with large symbol sets

### PR DESCRIPTION
**Pull request by ianhook**
The original symbol code required `injectSymbols` to be called separately from `makeSymbols` because `injectSymbols` deletes the Symbols page and recreates it on every call. This can take a while as seen in #230. 

I put the `injectSymbols` call in the render method, because this is the only time it needs to be called.

### CODE REVIEW

**Comment by alampros** [comment]
Thanks, @ianhook – this is SOO much faster!

I've added a branch to test this on the demo repo that I created for #230: https://github.com/alampros/react-sketchapp-svg-render-test/tree/test-pull-235

File: examples/symbols/src/my-command.js [hunk 1]
```diff
@@ -6,54 +6,51 @@ import {
   View,
   Image,
   makeSymbol,
+  injectSymbols,
```
Comment (alampros) [inline examples/symbols/src/my-command.js]: Can this be removed safely? It's throwing an eslint error.

**Comment by alampros** [comment]
This error is thrown on render if `makeSymbol` is never called:

```
TypeError: null is not an object (evaluating 'Object.keys(mastersNameRegistry)')
```

It looks like adding this check (from `makeSymbol`) to `injectSymbols` fixes it:
```js
  if (mastersNameRegistry === null) {
    getExistingSymbols();
  }
```

https://github.com/ianhook/react-sketchapp/blob/2a143bdf0a7c6649e35979d5c5fce976dc72a537/src/symbol.js#L75-L93

File: src/symbol.js [hunk 2]
```diff
@@ -72,36 +72,39 @@ export const getSymbolId = (masterName: string): string => {
...
+    renderLayers(Object.keys(layers).map(k => layers[k]), symbolsPage);
 
-  let notSymbolsPage = array.find(p => String(p.name()) !== 'Symbols');
-  if (!notSymbolsPage) {
-    notSymbolsPage = globalContext.document.addBlankPage();
+    let notSymbolsPage = array.find(p => String(p.name()) !== 'Symbols');
+    if (!notSymbolsPage) {
+      notSymbolsPage = globalContext.document.addBlankPage();
+    }
+    globalContext.document.setCurrentPage(notSymbolsPage);
```
Comment (mathieudutour) [inline src/symbol.js]: shouldn't we store the current page at the beginning of the `injectSymbols` method and go back to it instead getting the first one here?

**Comment by alampros** [comment]
What's involved in getting this merged and released?

**Comment by mathieudutour** [comment]
merged as part of #254

**Pull request closed without merging**